tasks/perform_orca_reconcile/perform_orca_reconcile.py

```diff
@@ -62,6 +72,7 @@ def task(
             None,
             user_engine,
         )
+        remove_job_from_queue(internal_report_queue_url, message_receipt_handle)
```

Comment:
May want to think about placement here or changing the except block here. Based on what you have here, we could have a case where everything is successful and DB is updated success but then we fail to remove it from the queue which would change it to a failure. 

I feel like this should be in its own try block separately maybe or have a retry associated  with it or a combination of the 2. Thoughts?

Reply:
Removed from try block. This does mean we can get some ghost queue entries if things go wrong.

Added basic retries. Added task https://bugs.earthdata.nasa.gov/browse/ORCA-399